cheat_sheets/pymechanical_cheat_sheet/pymechanical_script.py

```diff
@@ -69,15 +69,18 @@
 
 # BREAK BLOCK
 from ansys.mechanical.core import App
-
 app = App(version=241)
 print(app)
 # BREAK BLOCK
 
 # Extract the global API entry points (available from built-in Mechanical scripting)
 from ansys.mechanical.core import global_variables
 # Merge them into your Python global variables
-globals().update(global_variables(app))
+# Optional argument  for Enums without namespaces
+globals().update(global_variables(app),True)
+# Import Python modules shipped with Mechanical
+from ansys.mechanical.core.embedding import add_mechanical_python_libraries
+add_mechanical_python_libraries(241)
```

Comment:
Two comments - 
1. I don't see where these libraries are used.

2. better to make the version number a variable so that the same number is guaranteed to be used by `add_mechanical_python_libraries()` and `app = App()`